### PR TITLE
instructions to use your own deploy key in FAQ

### PR DESCRIPTION
**Pull request by philwinder**

### CODE REVIEW

File: site/faq.md [hunk 1]
```diff
@@ -49,3 +49,21 @@ Provide Flux with the registry credentials. See
...
+
+`kubectl delete secret flux-git-deploy`
+
+Then create a new one with your key:
+
+`kubectl create secret generic flux-git-deploy --from-file /path/to/id-rsa-flux`
+
+Now restart fluxd to re-read the k8s secret (if it is running):
+
+`kubectl delete -f deploy/flux-deployment.yaml ; kubectl apply -f deploy/flux-deployment.yaml`
```
Comment (sambooo) [inline site/faq.md]: Shouldn't it be enough to just run `kubectl delete $(kubectl get pod -o name -l name=flux)` and have kubernetes recreate the pod for us?

File: site/faq.md [hunk 2]
```diff
@@ -49,3 +49,21 @@ Provide Flux with the registry credentials. See
 Flux polls image registries every 5 minutes by default. You can change
 this, but beware that registries may throttle and even blacklist
 over-eager clients (like FLux in this scenario).
```
Comment (sambooo) [inline site/faq.md]: FLux -> Flux

File: site/faq.md (hunk 1 above)
Comment (philwinder) [inline site/faq.md]: Yep that should work.

File: site/faq.md [hunk 3]
```diff
@@ -48,4 +48,22 @@ Provide Flux with the registry credentials. See
...
+### How do I use my own deploy key?
+
+Flux uses a k8s secret to hold the git ssh deploy key. It is possible to
+provide your own.
+
+First delete the secret (if it exists):
+
+`kubectl delete secret flux-git-deploy`
+
+Then create a new secret named `flux-git-deploy` and using your key as the content of the secret:
```
Comment (sambooo) [inline site/faq.md]: Just noticed this. Should either say "..., using your key..." or "...and use your key..."

**Pull request merged**